adds new rules about  the robot placement technical challenge

fabiooju opened the pull request:
This is the new rules from robot placement technical challenge, including the procedure, evaluation, socring and tiebreak criteria. According to the league's objectives, this technical challenge will motivate the automatic positioning of the robots, aiming for compliance with the regular league.

fabiooju committed:
adds new rules about  the robot placement technical challenge

JoaoVictor0508 commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,59 @@
+
+1. Moving the robot to the desired position (each robot) — +5 
+2. Colliding with a robot during the placement — -1 (just one robot loses points) 
+3. Colliding with obstacles during the placement — -1 
```
Just to be clear, "obstacles" are the "opponent robots" that will be placed in the scenarios, right?

JoaoVictor0508 commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,59 @@
+For every scenario, the following criteria will be evaluated: 
+
+1. Moving the robot to the desired position (each robot) — +5 
+2. Colliding with a robot during the placement — -1 (just one robot loses points) 
```
```suggestion
2. Colliding with a team robot during the placement — -1

```

JoaoVictor0508 commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,59 @@
+2. Colliding with a robot during the placement — -1 (just one robot loses points) 
+3. Colliding with obstacles during the placement — -1 
+4. Colliding with one of field boundary — -1 
+
```
```suggestion

NOTE: A collision between two or more team robots will count as one collision, i.e., the collision will cost -1 for the team, not -1 for each robot.
```

JoaoVictor0508 commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,59 @@
+==== Scenarios 
+
+1. Kick-Off 
+2. Free Kick 
```
Are we going to consider the ball in the scenarios too? If so, the ball will be considered as an obstacle for the Scoring?

fabiooju committed:
adds text about collision rule score

a note specifying the collision rule score was added and the obstacles were specified

fabiooju committed:
Adds scenarios

ersaraujo commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,81 @@
+==== Scenarios 
+
+=== Scenario 1: Free Kick 1
+== Start
```
I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

ersaraujo commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,81 @@
+image::scenarios_robot_placement/free_kick1_goal.svg[Interactive,700,opts=interactive]
+
+=== Scenario 2: Kick Off
+== Start
```
I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

ersaraujo commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,81 @@
+image::scenarios_robot_placement/kick_off_goal.svg[Interactive,700,opts=interactive]
+
+=== Scenario 3: Free Kick 2
+== Start
```
I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

ersaraujo commented on 2024/2024-ssl-el-robot-placement.adoc:
```diff
@@ -0,0 +1,81 @@
+image::scenarios_robot_placement/free_kick2_goal.svg[Interactive,700,opts=interactive]
+
+Scenario 4: Penalty
+== Start
```
I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

fabiooju committed:
formatting and scenarios order changes

ersaraujo committed:
add pdf and html
